libs/hdf-converters/src/sarif-mapper.ts

```diff
@@ -122,25 +118,49 @@ export class SarifMapper extends BaseConverter {
               }
             },
             impact: {path: 'level', transformer: impactMapping},
-            code: '',
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string => {
+                return JSON.stringify(vulnerability, null, 2);
+              }
+            },
             results: [
               {
                 status: ExecJSON.ControlResultStatus.Failed,
                 code_desc: {
                   path: 'locations[0].physicalLocation',
                   transformer: formatCodeDesc
                 },
-                run_time: 0,
+
                 start_time: ''
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      meta_data: {
+        transformer: (data: Record<string, any>): Record<string, unknown> => {
+          data = _.omit(data, ['version']);
+          const keys = Object.keys(data['runs']);
+          keys.forEach((key) => {
+            data.runs[key] = _.omit(data.runs[key], ['results']);
+          });
+          return data;
+        }
+      },
+      raw: {
+        transformer: (
+          data: Record<string, unknown>
+        ): Record<string, unknown> | string => {
+          return this.passRaw ? data : '';
+        }
+      }
+    }
   };
-  constructor(sarifJson: string) {
+  constructor(sarifJson: string, passRaw = false) {
```

Comment:
i believe we were wanting to call it "withRaw" not "passRaw" to emphasize that the raw stuff would be coming with the rest of the hdf files.  consequently the saf cli would have an option for all mappers for "--with-raw" and then for the api based mappers specifically would have an option for "--raw" to just give the api data only.